CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Removed
 
 ### Fixed
+- Fix slow performance of FeatureFlag checks ([#17611](https://github.com/opensearch-project/OpenSearch/pull/17611))
```

Comment:
This should be moved to CHANGELOG-3.0.md

Reply:
Oh -- actually, I think I'm wrong. Since we released the 3.0-alpha, it looks like we've gone back to just CHANGELOG.md.